### ISSUE TITLE
Duplicate name warning while storing a result.

### ISSUE THREAD

**Issue by Xazax-hun** [Ericsson/codechecker#2510]
**Describe the bug**
Warning message when storing a run:
```
.../codechecker/build/CodeChecker/cc_bin/../lib/python2.7/codechecker_client/cmd/store.py:344: UserWarning: Duplicate name: '.../Fuchsia/fuchsia/third_party/glslang/glslang/Include/Common.h'
  zipf.write(f, os.path.join('root', f.lstrip('/')))
```

I see this warning multiple times, all of the warnings correspond to headers.

**CodeChecker version**
```
CodeChecker analyzer version:
---------------------------------------------------------------
Kind                 | Version                                 
---------------------------------------------------------------
Base package version | 6.12.0                                  
Package build date   | 2020-01-08T09:11                        
Git commit ID (hash) | 19c4caa6be4e103fe137f9bb45992cfb32b8260a
Git tag information  | 6.12                                    
---------------------------------------------------------------

CodeChecker web version:
------------------------------------------------------------------------
Kind                          | Version                                 
------------------------------------------------------------------------
Base package version          | 6.12.0                                  
Package build date            | 2020-01-08T09:11                        
Git commit ID (hash)          | 19c4caa6be4e103fe137f9bb45992cfb32b8260a
Git tag information           | 6.12                                    
Server supported API (Thrift) | 6.24                                    
Client API (Thrift)           | 6.24                                    
------------------------------------------------------------------------
```

(Btw, if one forgets to source the env, CodeChecker version will just not output anything and will not give any warnings/errors.)

**To Reproduce**
Fuchsia is enormous, so you probably do not want to try to reproduce it on that. 

**Expected behaviour**
No warnings seen. 

**Desktop (please complete the following information)**
 - OS: Debian
 - Browser: Chrome

**Comment by magnuspjonasson** [comment]
I came across the same 
"UserWarning: Duplicate name:"
message when doing
os.path.join(path_in_zip, existing_file))

Did you find an explanation?

**Comment by csordasmarton** [comment]
@magnuspjonasson How your store command looks like?

Could you please run this command in verbose mode (--verbose debug) and give us a full error log (only the context where this error log happens is enough)?
